src/guc.c

```diff
@@ -34,10 +34,10 @@ static const struct config_enum_entry telemetry_level_options[] = {
 	{ "off", TELEMETRY_OFF, false }, { "basic", TELEMETRY_BASIC, false }, { NULL, 0, false }
 };
 
-bool ts_guc_disable_optimizations = false;
+bool ts_guc_enable_optimizations = false;
```

Comment:
Should it be set to `true` here? It will correspond to before renaming.

Reply:
fixed